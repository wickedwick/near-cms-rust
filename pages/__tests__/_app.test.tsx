import React from 'react'
import { configure, shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import MyApp from "../_app";

configure({ adapter: new Adapter() });

const props: any = {
  Component: MyApp,
  pageProps: {},
}

describe('_app', () => {
  it('should render', () => {
    const wrapper = shallow(<MyApp {...props} />)
    expect(wrapper).toMatchSnapshot()
  });
})