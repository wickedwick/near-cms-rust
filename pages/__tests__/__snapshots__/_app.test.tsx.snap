// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_app should render 1`] = `ShallowWrapper {}`;
